docs/examples/vep.rst

```diff
@@ -7,7 +7,7 @@ tied to a specific reference genome.
 
 .. code-block:: shell
 
-   hailctl dataproc start cluster-name --vep GRCh37 --packages gnomad --requester-pays-allow-buckets gnomad-public-requester-pays
+   hailctl dataproc start cluster-name --vep GRCh37 --packages gnomad
```

Comment:
Sorry I havent followed which tables have moved but did the VEP data move as well? If it did, I think L14 needs to be updated/removed. If not, I think we should keep the requester pays options here since the examples below reference vep_or_lookup_vep which will need it.

Reply:
`--requester-pays-allow-buckets gnomad-public-requester-pays` because the resources would be read from that bucket by default.

The VEP context tables were already publicly available, but because the VEP context tables are loaded through the resources system, they'll be read from whichever source is configured. With this change, Google Cloud Public Datasets will be the default source, so ``--requester-pays-allow-buckets gnomad-public-requester-pays` is no longer needed unless someone chooses to change the resource source to `gs://gnomad-public-requester-pays`.

Reply:
Ooh, I misunderstood your comment. The note about VEP data being in a requester pays bucket refers to the data downloaded by Hail's VEP init scripts, not any of our data.

Reply:
No worries, it was worded poorly. Gotcha, thanks for clarifying!